src/hevm/src/EVM/SymExec.hs

```diff
@@ -544,13 +489,13 @@ verify solvers opts preState rpcinfo maybepost = do
         res <- checkSat solvers query
         pure (res, leaf)
       let cexs = filter (\(res, _) -> not . isUnsat $ res) results
-      pure $ if Prelude.null cexs then [Qed expr] else fmap toVRes cexs
+      pure $ if Prelude.null cexs then (expr, [Qed ()]) else (expr, fmap toVRes cexs)
```

Comment:
It may be helpful to return the unsimplified expression. Not sure what's best, to be honest. The unsimplified one can be very big, and it can always be simplified, but not the other way around :)

Reply:
hmmmm, I feel like we're usually going to want to display the simplified expression, and since we can pass `simp = False` in `VeriOpts` I would probably prefer to leave it as is for now?